portfolio/src/main/java/com/google/sps/servlets/PortfolioServlet.java

```diff
@@ -11,7 +14,23 @@
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse 
     response) throws IOException{
-           response.setContentType("text/html;");
-           response.getWriter().print("Scroll down to learn more!");
+           /*response.setContentType("text/html;");
+           response.getWriter().print("Scroll down to learn more!");*/
+           
+           //Array that contains hard-coded strings 
+           String facts[] = {"I am from Bulawayo in Zimbabwe!"," I am the last born child of 13 children!",
+           " My name is the name of a camera!" };
+           
+           String jsonResponse = convertToJsonUsingGson(facts);
+
+           response.setContentType("application/json;");
+           response.getWriter().print(jsonResponse);
     }
+
+    private String convertToJsonUsingGson(String[] stringArray) {
+    Gson gson = new Gson();
+    String json = gson.toJson(stringArray);
+    return json;
+  }
```

Comment:
You may insert a tab here so that your colleague can clearly see the code level.